src/font/lv_font_fmt_txt.c

```diff
@@ -182,6 +185,7 @@ bool lv_font_get_glyph_dsc_fmt_txt(const lv_font_t * font, lv_font_glyph_dsc_t *
     dsc_out->ofs_x = gdsc->ofs_x;
     dsc_out->ofs_y = gdsc->ofs_y;
     dsc_out->bpp   = (uint8_t)fdsc->bpp;
+    dsc_out->missing = false;
```

Comment:
What is the exact purpose of this flag?

Reply:
According to my observation, in freetype implementation, even a glyph is missing, it still can draw fallback glyph (tofu). So it's not enough to simply check if `lv_font_get_glyph_dsc_fmt_txt` can return true.

![image](https://user-images.githubusercontent.com/830358/142461543-a5e5ab6b-ef20-4b26-b746-3975e002045f.png)

Reply:
Got it. I think a name like `placeholder_glyph` would be more expressive. What do you think?

Reply:
I agree. I changed the name to `is_placeholder` for now.